### ISSUE TITLE
Default currency code based on time zone?

### ISSUE THREAD

**Issue by brashrebel** [brashrebel/don8#8]
Would be kind of neat to check the site's time zone and set the currency code default based on that value.

**Comment by chiaberry** [comment]
I saw this was in a pull request from last year, does it mean it's been solved/handled?

**Comment by brashrebel** [comment]
@chiaberry yes, this one's been completed. I just need to merge and deploy still.

**Comment by chiaberry** [comment]
thanks for replying @brashrebel !
